custom/icds_reports/management/commands/fix_past_data.py

```diff
@@ -0,0 +1,74 @@
+import os
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+
+
+from django.core.management.base import BaseCommand
+from corehq.util.argparse_types import date_type
+from django.db import connections, transaction
+from corehq.sql_db.connections import get_icds_ucr_citus_db_alias
+
+
+@transaction.atomic
+def _run_custom_sql_script(command, day=None):
```

Comment:
It looks like day is unused. That can be removed.

Reply:
updated!